app/templates/partials/stylesheets.jade

```diff
@@ -1,2 +1,2 @@
 link(rel='stylesheet', href='/styles/main.css')
-
+link(rel='stylesheet', href='/lib/horsey/dist/horsey.min.css')
```

Comment:
Always import the styles from third parties before your own. But in this case, it is even more preferred to use a sass import instead of directly linking.